entities/osv.bare64.main.cpi.entities.yml

```diff
@@ -0,0 +1,24 @@
+-
```

Comment:
Are all of these completely identical to the non-64-bit versions? If so, there's really no need for them.

Reply:
Resolved by runtime/arch split.